code/controllers/arrival_shuttle_controller.dm

```diff
@@ -0,0 +1,68 @@
+var/global/datum/arrival_shuttle_controller/arrival_shuttle
+
+/datum/arrival_shuttle_controller
```

Comment:
No, the actual controller is in arrivalShuttle.dm, a la the emergency shuttle prototype.